### PR TITLE
Feature/70 add tourbook ordering to custom settings

### LINKED ISSUE #70: Reverse order of tourbook entries
I think it is more comfortable to have the last entries first. The other solution would be to make it sortable.

### PR DESCRIPTION
**Pull request by paetz**
Resolves #70

### CODE REVIEW

File: app/src/main/res/values/setup.xml [hunk 1]
```diff
@@ -19,4 +19,7 @@
 <resources>
     <string name="app_name">YacGuide</string>
     <string name="app_name_long">Yet Another Climbing Guide</string>
+
+    <!-- default settings -->
+    <bool name="order_tourbook_chronologically">false</bool>
```
Comment (paetz) [inline app/src/main/res/values/setup.xml:24]: @chrgernoe What do you think about the best default value?

Comment (chrgernoe) [reply app/src/main/res/values/setup.xml:24]: Because I opened issue #70, false is my preferred value.

**Comment by paetz** [comment]
> Do we really need this button or would it be not more convenient to save the settings on-the-fly?

Yes, I thought a lot about this. We wouldn't need it if the settings had a parent activity (in which case we would store the data on returning). Since there is no parent activity, it is not clear for the user, how/if changes are stored (By pressing the phone's back-button? By changing to another activity?) That's why I chose the clear way.

**Comment by chrgernoe** [comment]
Understand. However, I don't know any app that has such a button. They save the settings on-the-fly and realize the layout as implemented in #128.

**Comment by paetz** [comment]
Ok, I'll check whether storing the data on onPause() callback. This is triggered whenever the activity looses focus.

**Pull request merged**